dataspace-connector/full/docker-compose.yml

```diff
@@ -56,9 +56,31 @@ services:
       - configmanager
     restart: always
 
+  keycloak:
+    depends_on:
+      - postgres
+    container_name: local_keycloak
+    environment:
+      DB_VENDOR: postgres
+      DB_ADDR: postgres
+      DB_DATABASE: connectordb
+      DB_USER: connector
+      DB_PASSWORD: 12345
+      KEYCLOAK_USER: admin
+      KEYCLOAK_PASSWORD: admin
+    image: jboss/keycloak:13.0.0
+    ports:
+      - "8180:8080"
+    restart: unless-stopped

```

Comment:
Docker:
- ‘Unless-stopped’ restarts the container only when any user executes a command to stop the container, **not when it fails because of an error**
- ‘Always’ restarts the container whether the stop is caused by an error, or is executed by a user, or if Docker is restarted.

Maybe _always_ would be a good choice here too?